src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,94 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api/api';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+interface Props {
+  addToList: AddToList;
+}
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ addToList }) => {
+  const [title, setTitle] = useState('');
+  const [foundedMovie, setFoundedMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState('');
+
+  const addMovie = () => {
+    addToList(foundedMovie);
+    setTitle('');
+    setError('');
```

Comment:
after movie is added to the list, form still shows it
<img width="1419" alt="Screenshot 2022-02-08 at 13 55 38" src="https://user-images.githubusercontent.com/57598825/152982260-d2b20ecf-285f-48eb-bbad-de1291a2dc42.png">